pkg/subctl/cmd/gather/cni.go

```diff
@@ -140,35 +140,35 @@ func gatherOVNResources(info *Info, networkPlugin string) {
 
 	// we check two different labels because OpenShift deploys with a different
 	// label compared to ovn-kubernetes upstream
-	pods, err := findPods(info.ClientSet, ovnMasterPodLabelOCP)
-	if err != nil || pods == nil || len(pods.Items) == 0 {
-		pods, err = findPods(info.ClientSet, ovnMasterPodLabelGeneric)
+	ovnMasterpods, err := findPods(info.ClientSet, ovnMasterPodLabelOCP)
```

Comment:
This was changed becasue golint complains of shadow variable declaration `pods` of package `pods`